src/FacebookLocales.js

```diff
@@ -60,6 +60,14 @@ const localesToNonStandardFacebookLocales = _(facebookVirtualLocales)
 	.value()
 
 export const bestFacebookLocaleFor = locale => {
+	// Normalize locale input format
+	if (locale.includes('-')) {
+		locale = locale.replace('-', '_');
+	}
+	if (locale.includes('_')) {
+		locale = locale.split('_')[0] + '_' + locale.split('_')[1].toUpperCase();

```

Comment:
This can be written more readable:

```js
const [language, territory] = locale.split('_');
locale = `${language}_${territory.toUpperCase()}`;
```

Reply:
I'm not sure which style is more acceptable, would like to wait review from its maintainer.